frontend/src/component/admin/users/LinkField/LinkField.tsx

```diff
@@ -5,19 +5,32 @@ import useToast from 'hooks/useToast';
 interface ILinkFieldProps {
     inviteLink: string;
     small?: boolean;
+    successTitle?: string;
+    errorTitle?: string;
 }
 
-export const LinkField = ({ inviteLink, small }: ILinkFieldProps) => {
+export const LinkField = ({
```

Comment:
Sth is off here. We have a generically named component LinkField that has invite link specific props and default values

Reply:
Yes. We have wrong tooltips on for example password reset. Nobody noticed. I'll have a fix in another PR